codechecker_lib/analyzers/analyzer_types.py

```diff
@@ -299,14 +289,11 @@ def build_config_handlers(args, context, enabled_analyzers, connection=None):
     for ea in enabled_analyzers:
         if ea == CLANG_SA:
             config_handler = __build_clangsa_config_handler(args, context)
-            analyzer_config_map[ea] = config_handler
-
         elif ea == CLANG_TIDY:
             config_handler = __build_clang_tidy_config_handler(args, context)
-            analyzer_config_map[ea] = config_handler
         else:
-            LOG.debug_analyzer('Not supported analyzer type. '
-                               'No configuration handler will be created.')
+            assert False, 'Analyzer types should have been checked already.'
```

Comment:
It does not work this way. See check_supported_analyzers which terminates the check when one of the analyzer is unsupported. And since the check right now is terminated there, in case this code is invoked with an unsupported analyzer, it is a programming error, so we should not continue running.